cases/aws_instance/run_instance_with_data_source_ami/main.tf

```diff
@@ -0,0 +1,51 @@
+resource "tls_private_key" "ssh" {
+  algorithm   = "RSA"
+}
+
+resource "aws_security_group" "additional_security_group" {
+  name        = "additional_security_group"
+  description = "additional_security_group"
+  vpc_id      = aws_vpc.test_vpc.id
```

Comment:
you need to add links to common resources from ./common/ folder to this folder if you use this resources.